libs/block-editor/src/lib/extensions/actions-menu.extension.ts

```diff
@@ -115,6 +115,8 @@ export const ActionsMenu = (viewContainerRef: ViewContainerRef) => {
     function onStart({ editor, range, clientRect }: SuggestionProps | FloatingActionsProps): void {
         suggestionsComponent = getSuggestionComponent(viewContainerRef);
         suggestionsComponent.instance.onSelection = (item) => {
+            // @ts-expect-error: As this point suggestion$ will be valid in the state.
+            range.to = range.to + editor.view.state.suggestion$.query.length;
```

Comment:
Here we can make a refactor to avoid the error:

`range.to = range.to + suggestionKey.getState(editor.view.state).query.length;`

In order to be able to use, we have to:

```js
const suggestionKey = new PluginKey('sugesstionPlugin');
....
    return Extension.create<FloatingMenuOptions>({
        name: 'actionsMenu',
        defaultOptions: {
            pluginKey: 'actionsMenu',
            suggestion: {
                char: '/',
                // Pass a pluginKey
                pluginKey: suggestionKey,
                allowSpaces: true,
                startOfLine: true,
             }
  ....
```